runtime/rasdump/dmpagent.c

```diff
@@ -2757,7 +2771,24 @@ runDumpAgent(struct J9JavaVM *vm, J9RASdumpAgent * agent, J9RASdumpContext * con
 		if (gotExclusive || (agent->dumpFn != doHeapDump)) {
 			agent->prepState = *state;
 			TRIGGER_J9HOOK_VM_DUMP_START(vm->hookInterface, vm->internalVMFunctions->currentVMThread(vm), label, detail);
+#ifdef J9ZTPF
+                        preHookInputParms.agentInfo = (jvmDumpAgentInfo *)agent;
+                        preHookInputParms.dumpPath = label;
+                        preHookOutputParms.dumpPath = label;
+                        preHookInputParms.context = (jvmDumpContext *)context;
+                        preHookInputParms.vm_args = (jvmInitArgs *)vm->vmArgsArray->actualVMArgs;
+                        cjvm_dumpagent_preHook(&preHookInputParms, &preHookOutputParms);
+                        label = preHookOutputParms.dumpPath;
+#endif
 			retVal = runDumpFunction( agent, label, context );
+#ifdef J9ZTPF
+                        postHookInputParms.agentInfo = (jvmDumpAgentInfo *)agent;
+                        postHookInputParms.dumpPath = label;
+                        postHookInputParms.context = (jvmDumpContext *)context;
+                        postHookInputParms.vm_args = (jvmInitArgs *)vm->vmArgsArray->actualVMArgs;
+                        cjvm_dumpagent_postHook(&postHookInputParms, &postHookOutputParms);
+                        cjvm_dumpagent_finished_hook((jvmDumpAgentInfo *)agent, label, (jvmDumpContext *)context);
+#endif
```

Comment:
Please use `defined(J9ZTPF)` and correct indentation.

Reply:
Done.